src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -1,5 +1,5 @@
 ﻿/// Contains a task which can be used to run dotnet CLI commands.
-module Fake.DotNet
+module Fake.DotNetCli
```

Comment:
@forki Is this OK, its a breaking change but the thing is new and it's currently colliding with my dotnetcore branch.
I really would like to have the `Fake.DotNet` namespace.